compiler/ballerina-lang/src/main/java/io/ballerina/projects/internal/ManifestBuilder.java

```diff
@@ -257,49 +255,53 @@ private PackageDescriptor getPackageDescriptor(TomlTableNode tomlTableNode) {
         String name;
         String version;
 
+        String errorMessage = "missing package information in Ballerina.toml, values derived as org: '"
+                + defaultOrg().value() + "', name: '" + defaultName(this.projectPath).value() + "' and version: '"
+                + defaultVersion().value().toString() + "'";
+
         if (tomlTableNode.entries().isEmpty()) {
-            reportDiagnostic(MISSING_PKG_INFO_ERROR,
+            reportDiagnostic(errorMessage,
                     ProjectDiagnosticErrorCode.MISSING_PKG_INFO_IN_BALLERINA_TOML.diagnosticId(),
                     DiagnosticSeverity.WARNING, tomlTableNode.location());
             return PackageDescriptor.from(defaultOrg(), defaultName(this.projectPath), defaultVersion());
         }
 
         TopLevelNode topLevelPkgNode = tomlTableNode.entries().get(PACKAGE);
         if (topLevelPkgNode == null || topLevelPkgNode.kind() != TomlType.TABLE) {
-            reportDiagnostic(MISSING_PKG_INFO_ERROR,
+            reportDiagnostic(errorMessage,
                     ProjectDiagnosticErrorCode.MISSING_PKG_INFO_IN_BALLERINA_TOML.diagnosticId(),
                     DiagnosticSeverity.WARNING, tomlTableNode.location());
             return PackageDescriptor.from(defaultOrg(), defaultName(this.projectPath), defaultVersion());
         }
 
         TomlTableNode pkgNode = (TomlTableNode) topLevelPkgNode;
         if (pkgNode.entries().isEmpty()) {
-            reportDiagnostic(MISSING_PKG_INFO_ERROR,
```

Comment:
I don't think we need this condition. It is covered by the org, name, and version checks below

Reply:
Removed in https://github.com/ballerina-platform/ballerina-lang/pull/36174/commits/48fba38888323c8a03144c32f77884571964360f